CONTRIBUTING.md

```diff
@@ -20,13 +20,12 @@ given according to the current bylaws of this group.
 
 # Merge & Access Policy
 
-All Editors, Coordinators and Sponsors of specifications in draft & review stage
+All Editors, Sponsors and WG members of specifications in draft & review stage
```

Comment:
Is this the only piece that's not accurate based on the bylaw changes since the PR was submitted?  Do we just need to drop `WG members` here?

I don't immediately see any issues with the remaining changes... or does @Jean85 's last comment from Jan mean the whole doc itself needs reworked?

Reply:
Is this the only piece that's not accurate based on the bylaw changes since the PR was submitted? Do we just need to drop `WG members` here?

I don't immediately see any issues with the remaining changes... or does @Jean85 's last comment from Jan mean the whole doc itself needs reworked?